packages/xo-server-perf-alert/src/index.js

```diff
@@ -524,11 +524,45 @@ ${monitorBodies.join('\n')}`
                   })
                 }
 
-                result.listItem = `  * ${result.objectLink}: ${
-                  result.value === undefined
-                    ? "**Can't read performance counters**"
-                    : result.value.toFixed(1) + typeFunction.unit
-                }\n`
+                const isManagementAgentDetected = (vm, guestMetrics) => {
+                  if (
+                    (vm.power_state !== 'Running' && vm.power_state !== 'Paused') ||
+                    vm.metrics === undefined ||
+                    guestMetrics === undefined
+                  ) {
+                    return
+                  }
+
+                  const { major, minor } = guestMetrics.PV_drivers_version
+                  const hasPvVersion = major !== undefined && minor !== undefined
+                  const pvDriversDetected = guestMetrics.PV_drivers_detected ?? hasPvVersion
+
+                  return pvDriversDetected || hasPvVersion || guestMetrics.other['feature-static-ip-setting'] === '1'
```

Comment:
My bad, I think I got confused with `pvDriversDetected`. The actual test for `managementAgentDetected` in XO is `hasPvVersion || guestMetrics.other['feature-static-ip-setting'] === '1'`.